Apps/WebClient/src/featuretoggleconfig.json

```diff
@@ -61,11 +61,8 @@
     },
     "dependents": {
         "enabled": true,
+        "timelineEnabled": true,
```

Comment:
What do you think of changing this to version: 1|2 or version: "card"|"timeline"?

Reply:
My understanding is that this setting will no longer be needed after the new functionality goes live.  At that time, the original dependent page can be removed, along with this setting, and the code will go back to checking just the dependents.enabled setting.